debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbStreamingChangeEventSource.java

```diff
@@ -209,7 +211,7 @@ private void readOplog(MongoClient primary, MongoPrimary primaryClient, ReplicaS
                 // In this situation if not document is available, we'll pause.
                 final Document event = cursor.tryNext();
                 if (event != null) {
-                    if (!handleOplogEvent(primaryAddress, event, event, 0, oplogContext, context)) {
+                    if (!handleOplogEvent(primaryAddress, event, event, 0, oplogContext, schema)) {
```

Comment:
The `context` wasn't used anywhere, so I removed it and added `schema`. Hence the replacement.